go.mod

```diff
@@ -14,7 +15,7 @@ require (
 	cloud.google.com/go v0.93.3 // indirect
 	cloud.google.com/go/kms v0.1.0 // indirect
 	cloud.google.com/go/pubsub v1.15.0
-	github.com/go-playground/universal-translator v0.17.0 // indirect
```

Comment:
git diff looks weird here but this is the result of `go mod tidy` and it runs so I'm not complaining